app/controllers/global_issue_templates_controller.rb

```diff
@@ -51,11 +50,11 @@ def edit
       @global_issue_template.safe_attributes = params[:global_issue_template]
       if @global_issue_template.save
         flash[:notice] = l(:notice_successful_update)
-        redirect_to :action => "show", :id => @global_issue_template.id
+        redirect_to action: "show", id: @global_issue_template.id
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)